test/suites/test_crud.py

```diff
@@ -0,0 +1,751 @@
+                'exception_input': {
+                    'args': [[],[]],
+                },
+                'exception_output': {
+                    'str': [],
+                },
+            },
+        },
+        'crud_truncate': {
+            'success': {
+                'correct_input': {
+                    'args': ['tester'],
+                },
+                'correct_output': {
+                    'scalar': True,
+                },
+            },
+            'error': {
+                'exception_input': {
+                    'args': ['no-such-space-name', {'timeout': 10}],
+                },
+                'exception_output': {
+                    'str': [
+                            r'"no-such-space-name" doesn\'t exist', 
+                        ],
+                },
+            },
+        },
+        'crud_stats': {
+            'success': {
+                'correct_input': {
+                    'args': [],
+                },
+                'correct_output': {
+                    'operations': [
+                            'insert', 'replace',
+                            'upsert', 'len',
+                            'delete', 'get',
+                            'select', 'borders',
+                            'update', 'count',
+                            'truncate',
+                        ],
+                },
+            },
+            'error': {
+                'exception_input': {
+                    'args': [],
+                },
+                'exception_output': {
+                    'str': [],
+                },
+            },
+        },
+    }
+
+    def _correct_operation_with_crud(self, testing_function, case, mode=None):
+        try:
+            if mode is not None:
+                resp = testing_function(
+                    *case['success']['correct_input']['args'],
+                    mode=mode
+                )
+            else:
+                resp = testing_function(
+                    *case['success']['correct_input']['args'],
+                )
+            if 'rows' in case['success']['correct_output']:
+                # Case for crud responce as tarantool.crud.CrudResult obj.
+                self.assertEqual(resp.rows, case['success']['correct_output']['rows'])
+            if 'scalar' in case['success']['correct_output']:
+                # Case for scalar value as crud responce, not tarantool.crud.CrudResult obj.
+                self.assertEqual(resp, case['success']['correct_output']['scalar'])
+            if 'operations' in case['success']['correct_output']:
+                # Case for statistics testing.
+                for operation in case['success']['correct_output']['operations']:
+                    self.assertEqual(operation in resp.spaces['tester'], True, 
+                        'Problem with finding a field with a statistic about operation ' 
+                            + operation)
+        except CrudModuleError as e:
```

Comment:
I think it's okay to let CrudModuleError just fall through and finish the test case with error.

Reply:
Done